src/main/webapp/feed.html

```diff
@@ -6,13 +6,24 @@
 <head>
```

Comment:
Aren't we deleted feed.html?

Reply:
Yes I just thought that thereference of messages in the nav bar was feed.html